test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -729,6 +729,84 @@ public void PackCommand_PackProject_SupportMultipleFrameworks()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
```

Comment:
Why it's only for Windows?
What prevents it running from MAC, Linux?

Reply:
Follow up issue to explore cross-platform integration tests https://github.com/NuGet/Home/issues/10426

Reply:
Update: Follow-up issue https://github.com/NuGet/Client.Engineering/issues/349